core/test/fake_client.go

```diff
@@ -105,6 +106,17 @@ func (f *fakeClientStruct) Build() *client.Client {
 	}
 }
 
+// GetRegisteredSchema ...
+func GetRegisteredSchema() *runtime.Scheme {
```

Comment:
This should be taken from the infrastructure package, or we can keep on core if it requires it, no problem.

Reply:
removed it